core/domain/wipeout_service.py

```diff
@@ -1492,18 +1492,27 @@ def _pseudonymize_blog_post_models(
     ] = blog_post_model_class.query(
         blog_post_model_class.author_id == user_id
     ).fetch()
-    blog_post_ids = {model.id for model in blog_post_models_list}
+    blog_related_model_ids = {model.id for model in blog_post_models_list}
 
     blog_post_summary_model_class = blog_models.BlogPostSummaryModel
     blog_post_summary_models: Sequence[
         blog_models.BlogPostSummaryModel
     ] = blog_post_summary_model_class.query(
         blog_post_summary_model_class.author_id == user_id
     ).fetch()
-    blog_post_ids |= {model.id for model in blog_post_summary_models}
+    blog_related_model_ids |= {model.id for model in blog_post_summary_models}
+
+    blog_author_details_model_class = blog_models.BlogAuthorDetailsModel
+    blog_author_details_model = blog_author_details_model_class.get_by_author(
+        user_id)
+    if blog_author_details_model:
```

Comment:
if blog_author_details_model if not None

Reply:
Done.